store/sdk/src/main/java/org/apache/carbondata/sdk/file/utils/SDKUtil.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Revert changes in this file as
these changes are not related to Binary, please keep in separate PR

Reply:
no, write binary will use this class to list binary file, like jpg